lib/ansible/modules/net_tools/basics/get_url.py

```diff
@@ -516,6 +516,9 @@ def main():
             module.fail_json(msg="Destination %s not readable" % (dest))
         checksum_dest = module.sha1(dest)
     else:
+        if not os.path.exists(os.path.dirname(dest)):
+            os.remove(tmpsrc)
+            module.fail_json(msg="Destination %s not exist" % (os.path.dirname(dest)))
```

Comment:
Please change message to `Destination %s does not exist`. (The word `does` is missing)